src/OPLS.js

```diff
@@ -212,18 +211,28 @@ export class OPLS {
     const tOrthCV = this.tOrthCV;
     const yHatCV = this.yHatCV;
     const m = this.model[nc - 1];
-    const XOrth = m.XOrth;
+    const XOrth = new Matrix(features.rows, features.columns);
```

Comment:
any reason it starts with an uppercase ? Variables are expected to start with lowercase.